### PR TITLE
DBAAS-941: upgrade to RDS controller v0.1.2

### PR DESCRIPTION
**Pull request by xieshenzh**
## Description

Upgrade to RDS controller v0.1.2

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA or in issue number have been completed
- [ ] Unit tests added that prove the fix is effective or the feature works 
- [ ] Documentation added for the feature
- [ ] all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: bundle/metadata/dependencies.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@ dependencies:
   - type: olm.package
     value:
       packageName: ack-rds-controller
-      version: ">=0.0.27 <=0.1.1"
+      version: ">=0.0.27 <=0.1.2"
```
Comment (redhatHameed) [inline bundle/metadata/dependencies.yaml:9]: What if we keep only single version 0.1.2?

Comment (xieshenzh) [reply bundle/metadata/dependencies.yaml:9]: DBaaS Operator won't upgrade or install until the ack-rds-controller is upgraded to 0.1.2

Comment (redhatHameed) [reply bundle/metadata/dependencies.yaml:9]: Looks like assigning range is best practice
https://olm.operatorframework.io/docs/concepts/olm-architecture/dependency-resolution/#best-practices

**Review by redhatHameed — APPROVED** [review]

**Comment by tchughesiv** [comment]
do we require `v0.1.2`? if so, should the dependency reflect that?

**Comment by tchughesiv** [comment]
@xieshenzh placing a hold on this. when ready please make this a larger PR which also upgrades to newer `rds-dbaas-operator` release which has the exact same ack dependency set. If there's not alignment, the install will break/conflict.

**Comment by xieshenzh** [comment]
> @xieshenzh placing a hold on this. when ready please make this a larger PR which also upgrades to newer `rds-dbaas-operator` release which has the exact same ack dependency set. If there's not alignment, the install will break/conflict.

Added `rds-dbaas-operator` version upgrade.

**Review by tchughesiv — APPROVED** [review]
lgtm

**Review by jeremyary — APPROVED** [review]

**Pull request merged**